src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
###AWATDZxn57qISpJ8hDW_###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_22@6
Complete the task associated to this TODO comment.

Reply:
simulating user reply

Reply:
simulating bot reply

Reply:
another user reply